calico/src/main/scala/calico/html/Prop.scala

```diff
@@ -146,37 +147,37 @@ private trait PropModifiers[F[_]](using F: Async[F]):
     Modifier.forSignalResource[F, Any, OptionSignalResourceModifier[F, Any, Any], Option[Any]](
       _.values) { (m, n) => setPropOption(n, m.name, m.encode) }
 
-final class EventProp[F[_], E, A] private[calico] (key: String, pipe: Pipe[F, E, A]):
+final class EventProp[F[_], A] private[calico] (key: String, pipe: Pipe[F, Any, A]):
```

Comment:
Bye `E`, hope we don't miss you.

Reply:
Actually I think it wasn't needed even before, but I only realized when I did this PR.

Reply:
Yes, I think this is an orthogonal change. It would bother me because I like to pretend `Any` doesn't exist, but I see that we need that later on either way, because it helps us saving evil allocs ;)